packages/icons/icons.json

```diff
@@ -4198,5 +4198,89 @@
         "tags": "data, type, format，punctuation, grammar",
         "group": "miscellaneous",
         "codepoint": 62296
+    },
+    {
+        "displayName": "Two column view",
+        "iconName": "two-column-view",
```

Comment:
I think it would be better if all these new icons started with a `layout-` prefix so that they could be auto-completed / auto-suggested in a more intuitive way. Then I think you could also drop the `-view` suffix. Something like this:

```
layout-two-columns
layout-two-rows
layout-three-columns
layout-three-rows
layout-bottom-row-two-tiles
...
```

Note that you'll also have to update all the `.svg` file names after you edit `"iconName"` fields in this JSON file.

Reply:
make sense! I updated the name to be `layout-*`. ( sorry about staling it for so long XD. ready for another round of review!